pkg/ddevapp/codeigniter.go

```diff
@@ -0,0 +1,197 @@
+package ddevapp
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ddev/ddev/pkg/archive"
+	"github.com/ddev/ddev/pkg/fileutil"
+	"github.com/ddev/ddev/pkg/output"
+	"github.com/ddev/ddev/pkg/util"
+)
+
+// createCodeIgniterSettingsFile creates/updates the .env file for CodeIgniter 4
+func createCodeIgniterSettingsFile(app *DdevApp) (string, error) {
+	envFilePath := filepath.Join(app.AppRoot, ".env")
+
+	// If .env already exists, leave it alone
+	if fileutil.FileExists(envFilePath) {
+		output.UserOut.Printf("CodeIgniter .env file already exists at %s", envFilePath)
+		return envFilePath, nil
+	}
+
+	// Try to copy from env or .env.example
+	envExamplePath := filepath.Join(app.AppRoot, "env")
+	if !fileutil.FileExists(envExamplePath) {
+		envExamplePath = filepath.Join(app.AppRoot, ".env.example")
+	}
```

Comment:
Is this some standard approach for CodeIgniter?

I've never heard of using `env` without dot.